--- __main__.py	2023-07-17 10:06:16.000000000 +0200
+++ __main__.py	2023-07-24 22:24:07.143239720 +0200
@@ -1,6 +1,6 @@
 import os
 
-import pkg_resources
+from importlib.metadata import packages_distributions, version
 import warnings
 
 import click
@@ -28,9 +28,12 @@
 
 
 def print_version():
-    for pkg in pkg_resources.working_set:
-        if pkg.key.startswith('riptide-'):
-            print(f"{pkg.key:>30}: {pkg.version}")
+    pkgs = packages_distributions()
+    if 'riptide' not in pks.keys():
+        print("Couldn't find riptide module in top-level modules:", pkgs.keys())
+        exit(1)
+    for pkg in packages_distributions()['riptide']:
+            print(f"{pkg:>30}: {version(pkg)}")
 
 
 @click.group(
