--- setup.py	2023-05-09 17:33:13.000000000 +0200
+++ setup.py	2023-07-24 22:32:47.146736091 +0200
@@ -12,7 +12,7 @@
     name='configcrunch',
     version='1.0.4',
     packages=find_packages(),
-    package_data={"configcrunch": ["py.typed", "*.pyi"]},
+    package_data={"configcrunch": ["py.typed", "*.pyi"], "configcrunch.tests.fixtures": ["**/*.yml", "**/*.yaml"]},
     rust_extensions=[RustExtension(f"configcrunch._main", binding=Binding.PyO3)],
     description='Configuration parser based on YAML-Files with support for variables, overlaying and hierarchies',
     long_description=long_description,
