--- loader.py	2023-07-17 12:33:21.000000000 +0200
+++ loader.py	2023-07-24 22:24:07.173239730 +0200
@@ -1,4 +1,4 @@
-import pkg_resources
+from importlib.metadata import entry_points
 
 from riptide.plugin.loader import load_plugins
 
@@ -10,7 +10,7 @@
     # Look up package entrypoints for engines
     engines = {
         entry_point.name:
-            entry_point.load() for entry_point in pkg_resources.iter_entry_points(ENGINE_ENTRYPOINT_KEY)
+            entry_point.load() for entry_point in entry_points(group=ENGINE_ENTRYPOINT_KEY)
     }
     if engine_name in engines:
         instance = engines[engine_name]()
